src/redux/features/observations/observationsSelectors.ts

```diff
@@ -58,8 +58,8 @@ export const selectPlantingSiteObservationsResults = createCachedSelector(
   }
 )((state, plantingSiteId: number, status?: ObservationState[]) => `${plantingSiteId}_${status?.join(',')}`);
 
-export const selectPlantingSiteAdHocObservationsResults = createCachedSelector(
-  (state: RootState, plantingSiteId: number) => selectAdHocObservationsResults(state),
+export const selectPlantingSiteAdHocObservationResults = createCachedSelector(
+  (state: RootState, plantingSiteId: number) => selectadHocObservationResults(state),
```

Comment:
Why changing the camel case on selectAdHocObservationsResults to selectadHocObservationResults?